cmd/minikube/cmd/mount.go

```diff
@@ -51,7 +51,7 @@ const (
 	nineP                     = "9p"
 	defaultMount9PVersion     = "9p2000.L"
 	mount9PVersionDescription = "Specify the 9p version that the mount should use"
-	defaultMountGID           = "docker"
+       defaultMountGID           = "root"
```

Comment:
Why do we change mount command user? how is this related to changing ssh key algorithm?